src/transformers/utils/hub.py

```diff
@@ -491,7 +492,10 @@ def cached_file(
             return None
 
         raise EnvironmentError(f"There was a specific connection error when trying to load {path_or_repo_id}:\n{err}")
-
+    except HFValidationError as e:
+        raise EnvironmentError(
+            f"Incorrect path_or_model_id: {path_or_repo_id}. Please provide either the path to a local folder or the repo_id of a model on the Hub. {e}"
+        ) from e
```

Comment:
I am not sure we should copy the HfValidationError message into the EnvironmentError one. I feel that `Repo id must use alphanumeric chars or '-', '_', '.', '--' and '..' are forbidden, '-' and '.' cannot start or end the name, max length is 96: '.gpt2'.` is very long and doesn't provide much value to the user. What's really important is `Please provide either the path to a local folder or the repo_id of a model on the Hub.` (that you've added).

The HfValidationError message is useful for example when creating or moving a repository (in which case repo_id validation is necessary).


(also, since you are raising "from e", the error message will still be in the stacktrace of the user if it's really important)

```suggestion
        raise EnvironmentError(
            f"Incorrect path_or_model_id: {path_or_repo_id}. Please provide either the path to a local folder or the repo_id of a model on the Hub."
        ) from e
```

Reply:
OK! Thanks for the review 😉